errai-data-binding/src/main/java/org/jboss/errai/databinding/client/api/IgnoreBinding.java

```diff
@@ -0,0 +1,19 @@
+package org.jboss.errai.databinding.client.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.jboss.errai.databinding.rebind.BindableProxyGenerator;
+
+/**
+ * Indicates that method or field should be ignored by the {@link BindableProxyGenerator}

```

Comment:
This javadoc is too implementation specific. Most of our users don't know what the `BindableProxyGenerator` is (and that's probably a good thing!).

From my reading of the code, this annotation:
* When put on a bean accessor (getter or setter) causes Errai to ignore that accessor. If a properties accessors are ignored, it will not be considered a binable property.
* When put on a non-accessor, causes Errai to not proxy that method. Normally Errai proxies non-accessor methods so that we can check for model updates that happen outside of getter and setter calls.

Could you please update the javadoc to describe this behaviour?